tests/integration_tests/test_mongo_task.py

```diff
@@ -98,6 +98,7 @@ def test_combined_erasure_task(
     }
 
 
+@pytest.mark.integration_mongodb
```

Comment:
This is a great spot. I'm not sure whether the test would run in the current format if it was marked with both (or it could run twice)

Reply:
Just looked at changing this — was this commend in reference to the test above `test_combined_erasure_task`?

Reply:
yes! i couldn't comment higher than this line

Reply:
I remember now why I didn't do this. Right now the test only has `@pytest.mark.integration`. If we add either of the others, the test will fail if somebody runs only `make pytest-integration datastores=mongodb` or `make pytest-integration datastores=postgres` because the test will be triggered and one set of infrastructure will be unavailable. Pytest doesn't provide a way to combine markers and only run the test if both are present.

This is an issue though as this new script no longer uses `-m integration` in favour of running each specific mark individually. I'll have to add the generic one back in somehow.

An alternative could be `@pytest.mark.integration_postgres_and_mongo`, but that seems to defeat the point of splitting these out in the first place, and there's no obvious way we'd trigger it.

Reply:
In short, a combined test that extends multiple databases needs to bring up all of the databases?

Reply:
Not technically of course, it's just hard to specify the relationship between which DBs to bring up and which tests to execute otherwise